### ISSUE TITLE
(Feature Request) Import/ Exports notes

### ISSUE THREAD

**Issue by ekirchman** [Abdallah-Abdelazim/My-Notes#6]
Adding the feature to export and import notes would be really helpful. Such as when you need to back up notes or such

**Comment by Abdallah-Abdelazim** [comment]
I agree with you that import/export data would be a great feature. I've had that feature in mind when I started this project.
But when I started with this project I was only a beginner in Android. Also, I built it with old techniques (I used Java not Kotlin, didn't use Room for the database or any of the new Architecture components, all code in the Activity with no architecture pattern used, etc...).
So continuing to add features in this project as it is won't be that effective (keeping in mind that this project is for educational purposes).
I have been planning for a while on putting the current code on a branch (for archival) and rewriting the project from scratch with all the great things introduced in Android in the last two years. But I've been procrastinating ever since 😅.
I promise that I'll get back to this project adding your requested feature & also others.
Thank you for your suggestions ❤️.
